Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -140,16 +192,30 @@ private enum Wires
 
             /// <see cref="MainPower"/>
             BackupPower,
+
+            /// <summary>
+            /// Pulsing causes for bolts to toggle (but only raise if power is on)
+            /// Cutting causes Bolts to drop
+            /// Mending does nothing
+            /// </summary>
+            Bolts,
+
+            /// <summary>
+            /// Pulsing causes light to toggle
+            /// Cutting causes light to go out
+            /// Mending causes them to go on again
+            /// </summary>
+            BoltLight,
         }
 
         public void RegisterWires(WiresComponent.WiresBuilder builder)
         {
             builder.CreateWire(Wires.MainPower);
             builder.CreateWire(Wires.BackupPower);
-            builder.CreateWire(1);
-            builder.CreateWire(2);
-            builder.CreateWire(3);
+            builder.CreateWire(Wires.Bolts);
+            builder.CreateWire(Wires.BoltLight);
             builder.CreateWire(4);
+            builder.CreateWire(5);
             /*builder.CreateWire(5);
```

Comment:
Should delete this line from the comment since this is no longer the next free wire.